ajax/libs/typeboost.css/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "typeboost.css",
+  "filename": "typeboost.min.css",
+  "version": "0.1.8",
+  "description": "Typeboost.css is a set of CSS rules to ensure readable, scalable and responsive typography. No dependencies. REMs are used for font sizes, paddings, and margins. Wrap your HTML content with class .col and you are done.",
+  "homepage": "https://englishextra.github.io/libs/typeboost.css/",
+  "keywords": [
+    "typeboost.css"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/englishextra/typeboost.css.git",
+    "fileMap": [
+      {
+        "basePath": "css",
```

Comment:
Sorry for making you confused. We will not ask author to change the file structure of their repo. We just want to keep the file structure as the same as upstream.